source/style-guide/style/code-examples.txt

```diff
@@ -60,113 +60,60 @@ or output examples:
-
-#. Create a VM that's running Docker.
-
-   .. code-block:: sh
+- Set code blocks that contain only example output to 
+  ``:copyable: false``. 
 
-      $ docker-machine create --driver virtualbox test
-
-   The ``--driver`` flag indicates what type of driver the machine runs
-   on. In this case, ``virtualbox`` indicates that the driver is Oracle
-   VirtualBox. The final argument in the command gives the VM a name,
-   in this case, ``test``.
-
-   The output should look as follows:
-
-   .. code-block:: sh
-
-      Creating VirtualBox VM...
-      Creating SSH key...
-      Starting VirtualBox VM...
-      Starting VM...
-      To see how to connect Docker to this machine, run:
-      docker-machine env test
-
-#. Run docker-machine ls again to see the VM that you created.
-
-   The output should look as follows:
+The following examples illustrate many of these guidelines:
 
-   .. code-block:: sh
+Configure the MongoDB CLI
+-------------------------
 
-      NAME ACTIVE DRIVER STATE URL SWARM
-      test virtualbox Running tcp://192.168.99.101:237
+#. Create a profile.
 
-Run the application
--------------------
+   This command creates a profile with a custom name in the MongoDB CLI
+   Configuration File. To use this profile, you must specify it with the
+   ``--profile`` flag in every MongoDB CLI command.
 
-#. Run a container from the image. The application code uses the
-   environment variables that you defined to connect to the MongoDB
-   container.
+   Run the ``mongocli config`` command:
 
    .. code-block:: sh
+   
+      mongocli config  --profile <profile-name>
 
-      $ docker run --detach \
-        --env MONGO_HOST=$MONGO_HOST \ env MONGO_PORT=$MONGO_PORT \ env
-        --MONGO_SSL=$MONGO_SSL \ env MONGO_DATABASE=$MONGO_DATABASE \ env
-        --MONGO_USER=$MONGO_USER \ env MONGO_PASSWORD=$MONGO_PASSWORD \ publish
-        --5000:5000 \
-        guestbook-mongo:1.0
+#. Enter your Programmatic API Key and Project ID.
 
-#. View the status of the container by using the ``--latest``
-   parameter.
+   When prompted, enter the Public API Key, Private API Key, and Project ID for your Atlas deployment.
 
    .. code-block:: sh
+      :copyable: false
 
-      $ docker ps --latest
-
-The status of the container should begin with ``Up``.
+      ? Public API Key: abcdef
+      ? Private API Key: [? for help] ************************************
+      ? Project ID [optional]: 7ehf04abc10defb66c7d1234c
 
-Remove the Containers Already using the Port
---------------------------------------------
+   If you don’t save your project ID in your profile, you must specify the ``--projectId`` option every time you run a MongoDB CLI command.
 
-#. To identify the containers that are using the port, run the
-   following command, changing ``<port>`` to the port number that you
-   want to use.
+#. Verify your profile.
+   
+   View the configuration file or run the mongocli config describe command to verify your profile.
```

Comment:
Question: should some part of "mongocli config describe" be differentiated as a command with monospace or bold here?